### PR TITLE
Release 0.0.1

### PR DESCRIPTION
**Pull request by OptimumCode**

### CODE REVIEW

**Comment by OptimumCode** [comment]
@Nikita-Smirnov-Exactpro Could you please take a look so we have the first release

File: core/src/main/kotlin/com/exactpro/th2/read/db/core/impl/DataBaseServiceImpl.kt [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+            while (resultSet.next()) {
+                if (columns == null) {
+                    columns = resultSet.extractColumns()
+                }
+                emit(resultSet.transform(columns, associatedMessageType))
+            }
+        }.onCompletion { reason ->
+            reason?.also { LOGGER.warn(it) { "query $queryId completed with exception for $dataSourceId source" } }
+            LOGGER.trace { "Closing connection to $dataSourceId" }
+            runCatching { connection.close() }.onFailure { LOGGER.error(it) { "cannot close connection for $dataSourceId" } }
```
Comment (Nikita-Smirnov-Exactpro) [inline core/src/main/kotlin/com/exactpro/th2/read/db/core/impl/DataBaseServiceImpl.kt:90]: Do we use connection pool ?

File: gradle.properties [hunk 2]
```diff
@@ -0,0 +1,3 @@
+vcs_url=https://github.com/th2-net/th2-grpc-read-db
+release_version=0.0.0
```
Comment (Nikita-Smirnov-Exactpro) [inline gradle.properties:2]: Change to not 0 value

File: gradle/wrapper/gradle-wrapper.properties [hunk 3]
```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.5.1-bin.zip
```
Comment (Nikita-Smirnov-Exactpro) [inline gradle/wrapper/gradle-wrapper.properties:3]:
```suggestion
distributionUrl=https\://services.gradle.org/distributions/gradle-7.6-bin.zip
```

**Review by Nikita-Smirnov-Exactpro — APPROVED** [review]

**Pull request merged**